src/main/resources/config.properties

```diff
@@ -22,4 +22,5 @@
 # NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 #
-API_VERSION=32.0
\ No newline at end of file
+API_VERSION=32.0
```

Comment:
agree that they are separate modules. However, they all interact with our platform, and it's a best practice to maintain that parity. Please chose a API version that is consistent across pom.xml dependencies and such calls within the code